src/Xamarin.Android.Tools.Bytecode/Tests/ParameterFixupTests.cs

```diff
@@ -30,27 +30,22 @@ public void XmlDeclaration_FixedUpFromDocumentation()
 		}
 
 		[Test]
-		public void XmlDeclaration_FixedUpFromApiXmlDocumentation()
+		public void XmlDeclaration_FixedUpFromApiXmlDocumentation ()
 		{
 			string tempFile = null;
 
-			try
-			{
-				tempFile = Path.GetTempFileName();
-				File.WriteAllText(tempFile, LoadString("ParameterFixupApiXmlDocs.xml"));
+			try {
+				tempFile = LoadToTempFile ("ParameterFixupApiXmlDocs.xml");
 
-				AssertXmlDeclaration("Collection.class", "ParameterFixupFromDocs.xml", tempFile, Bytecode.JavaDocletType._ApiXml);
-			}
-			catch (Exception ex)
-			{
-				try
-				{
-					if (File.Exists(tempFile))
-						File.Delete(tempFile);
+				AssertXmlDeclaration ("Collection.class", "ParameterFixupFromDocs.xml", tempFile, Bytecode.JavaDocletType._ApiXml);
+			} catch (Exception ex) {
+				try {
+					if (File.Exists (tempFile))
```

Comment:
This shouldn't be in the `catch` block, this should be in a `finally` block, so that we *always* delete `tempFile`. (Looks like this was a bug in the original code.)